### DISCUSSION TITLE
Can't start webone

### DISCUSSION THREAD

**Discussion by Vulpovile** [atauenis/webone#60] [Q&A] [answered]
Cannot start server: The process cannot access the file because it is being used
 by another process!

occurs any time I try starting it.
I ran with admin privileges. 
System:

Windows 7 Professional (64 bit)

**Comment by atauenis** [comment]
Seems that the system is broken. There is a problem on .NET Core Runtime, or Microsoft's HTTP.SYS driver (included in Windows 7).

Probably things which may solve this problem:
```
1. CMD with admin rights:
netsh http add urlacl url=http://*:8080/ user=Vulpovile
                                              (^^^^^^^^ local system user account name)
netsh advfirewall firewall add rule name=HTTP dir=in action=allow protocol=TCP localport=8080
```
2. Try to install Microsoft Internet Information Services.
3. Try to unload antivirus if any is installed.
4. If all don't help, it is need to reinstall Windows 7.

Also this may be caused by incorrect file system access control rights (an read-only directory or something).

**Comment by Vulpovile** [comment, marked as answer]
Found the problem:
Apache! Did not notice it has started
It would be nice if the error message could be a bit more descriptive
